From d153f384e3d3e5ca512e3237421b51960b1a2847 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Fri, 21 Feb 2025 14:58:25 +0100
Subject: [PATCH 31/35] arm64: dts: lx2160a-cex6-evb: replace
 fsl,unmanaged-lanes dt property

The fsl,unmanaged-lanes functionality has been replaced by the status of
lane-specific phy nodes.

Update board description accordingly.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 .../dts/freescale/fsl-lx2160a-cex6-evb.dts    | 21 ++++++++++++++++---
 1 file changed, 18 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex6-evb.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex6-evb.dts
index 2258ab3c04b8..97a7597c252e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex6-evb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex6-evb.dts
@@ -218,9 +218,24 @@ &pcs_mdio6 {
 	status = "okay";
 };
 
-&serdes_1 {
-	/* disable driver for lanes A-D */
-	fsl,unmanaged-lanes = <0x0f>;
+&serdes_1_lane_a {
+	/* lane is used for pci */
+	status = "disabled";
+};
+
+&serdes_1_lane_b {
+	/* lane is used for pci */
+	status = "disabled";
+};
+
+&serdes_1_lane_c {
+	/* lane is used for pci */
+	status = "disabled";
+};
+
+&serdes_1_lane_d {
+	/* lane is used for pci */
+	status = "disabled";
 };
 
 &usb0 {
-- 
2.43.0

